lib/src/system_cache.dart

```diff
@@ -35,8 +35,14 @@ class SystemCache {
     if (Platform.environment.containsKey('PUB_CACHE')) {
       return Platform.environment['PUB_CACHE'];
     } else if (Platform.isWindows) {
+      // If a cache dir already exists in %APPDATA%, use it, else default to %LOCALAPPDATA%
```

Comment:
nit, Can we briefly explain that we prefer `%APPDATA%`  if it already exists because older pub versions created the pub-cache here.
And maybe state that using `%LOCALAPPDATA%` is better because it's not synchronized.